components/script/dom/idbrequest.rs

```diff
@@ -126,6 +150,21 @@ impl RequestListener {
                         warn!("Error reading structuredclone data");
                     }
                 },
+                IdbResult::Values(serialized_values) => {
+                    let mut array: Vec<JSVal> = Vec::with_capacity(serialized_values.len());
+                    for serialized_data in serialized_values {
+                        let data = StructuredSerializedData {
+                            serialized: serialized_data,
+                            ..Default::default()
+                        };
```

Comment:
This needs to be updated to use bincode::deserialize to obtain the StructuredSerializedData.